build/Makefile

```diff
@@ -413,6 +414,10 @@ build-licenses:
 build-required-src-dist:
 	docker run --rm $(common_mounts) $(build_tag) $(mount_path)/build/build-required-src-dist.sh
 
+# Create docker builder used in creating arm64 images
```

Comment:
I'd love to see a comment/summary here on why the builder is necessary, maybe with pointing to one or more articles on the subject - so it's easy to look back and work out what is going on.